source/extensions/extensions_build_config.bzl

```diff
@@ -173,6 +173,12 @@ EXTENSIONS = {
     "envoy.tracers.xray":                               "//source/extensions/tracers/xray:config",
     "envoy.tracers.skywalking":                         "//source/extensions/tracers/skywalking:config",
 
+    #
+    # Listener
+    #
+
+    "envoy.listener.quic":                              "//source/extensions/quic_listeners/quiche:quic_factory_lib",
+
```

Comment:
@danzh2010 wondering if there is a reason behind creating a new category of `envoy.listener` here vs. using the existing category of `envoy.filter.listener` used by 5 extensions at https://github.com/envoyproxy/envoy/blob/main/source/extensions/extensions_build_config.bzl#L91-L103

Reply:
`envoy.filter.listener` are listener filters, but `envoy.listener` is supposed to be implementation of listener.

Reply:
That makes sense. Thanks